cmd/podman/diff.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"fmt"
 	"github.com/containers/libpod/cmd/podman/cliconfig"
-	"github.com/containers/libpod/cmd/podman/formats"
+	"github.com/containers/buildah/cmd/formats"
```

Comment:
Can we use them as a buildah/pkg/formats?

Reply:
:ok_hand: